frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -618,13 +617,13 @@ function useWMGQueryRequestBody(options = { includeAllFilterOptions: false }) {
 
     return {
       filter: {
-        dataset_ids: datasets,
-        development_stage_ontology_term_ids: developmentStages,
-        disease_ontology_term_ids: diseases,
-        ethnicity_ontology_term_ids: ethnicities,
+        dataset_ids: datasets ?? [],
```

Comment:
I think it would be worthwhile determining the root cause of the `undefined` values here (and the need for the nullish coalescing); the answer will likely be valuable knowledge for the team. @atarashansky, can you take another look at determining where the `undefined` values are coming from?

Reply:
bump!

Reply:
Long story short - I haven't been able to figure it out yet.

Reply:
@MillenniumFalconMechanic I found the root cause. It turns out they were getting initialized to undefined in the reducer. I am setting it now to empty arrays by default and removed the nullish coalescing.